action.yml

```diff
@@ -8,6 +8,9 @@ inputs:
   app_name:
     description: "Name to use for generated application artifacts"
     required: true
+  create_pull_secret_from:
+    description: "Auth file from which to create pull secret, accepted values are 'docker' and 'podman'"
```

Comment:
I don't think this is a very clear description. how about:

Registry credentials file to use to create a pull secret. Set this to "docker" or "podman" depending on which tool you used to log in.